configs/examples/atari-adversary-curriculum.json

```diff
@@ -0,0 +1,68 @@
+[
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 1,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 2,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 4,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 8,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 16,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 32,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 64,
+        "win_rate": 0.75
+    },
+    {
+        "name": "v9-s71m.bin.gz",
+        "max_visits_victim": 128,
+        "win_rate": 0.75
+    },
+    {
+        "_comment": "in the paper run we increased the win rate threshold to 0.9 here instead",
```

Comment:
Historically we've seen it be significantly slower to reach 90% win rate compared to 75% win rate. The cost between 256 victim visits and 512 victim visits isn't that big and I think it's better to move onto higher visits slightly earlier in training to train on harder opponents + since attacks at lower visits don't necessarily carry over to high visits
But for ViT-adversary I kept the win rate threshold at 0.9 for 256 visits since against the ViT there's high transfer to higher visits